frontstage/controllers/auth_controller.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import requests
+from flask import current_app as app
+from structlog import wrap_logger
+
+from frontstage.common.utilities import obfuscate_email
+from frontstage.exceptions.exceptions import ApiError, AuthError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def sign_in(username, password):
+    bound_logger = logger.bind(email=obfuscate_email(username))
+    bound_logger.info('Attempting to sign in')
+
+    url = f"{app.config['AUTH_URL']}/api/v1/tokens/"
+    data = {
+        'username': username,
+        'password': password,
+    }
+    headers = {
+        'Accept': 'application/json',
+        'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
+    }
+    response = requests.post(url, headers=headers, auth=app.config['BASIC_AUTH'], data=data)
+
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        auth_error = response.json().get('detail', '')
```

Comment:
Looking at the ras-rm-auth-service, I realise I was wrong about this bit.  We need to handle the 401 error (to log a warning for a failed login attempt), but we also need the non-401 error incase something more serious is happening when we try to connect to the auth service. 
How we have it here, if the service was returning 500s, we'd still just be logging warnings when something was actually on fire.